npc/re/instances/GhostPalace.txt

```diff
@@ -0,0 +1,975 @@
+//===== rAthena Script =======================================
+//= Ghost Palace
+//===== Description: =========================================
+//= [Walkthrough Conversion]
+//= Ghost Palace Instance
+//===== Changelogs: ==========================================
+//= 1.0 First version. [Capuche]
+//============================================================
+
+dali02,43,129,5	script	Unpleasent Royal Guard#	4_M_SAKRAYROYAL,{
+	if (BaseLevel < 120) {
+		mes "[Unpleasent Royal Guard]";
+		mes "Althought I need help with a task, you are not strong enough to assist me, noob~!";
+		next;
+		mes "[Unpleasent Royal Guard]";
+		mes "Come back after when you have done some training and are at least level 120!";
+		close;
+	}
+	.@party_id = getcharid(1);
+	if (getcharid(0) != getpartyleader(.@party_id,2) || .@party_id < 1) {
+		mes "[Unpleasent Royal Guard]";
+		mes "I can only open the secret passage for party leaders. Please have your party leader talk to me.";
+		close;
+	}
+	switch( checkquest(1261,PLAYTIME) ) {
+	case -1:
+		break;
+	case 0:
+	case 1:
+		mes "[Unpleasent Royal Guard]";
+		mes "Have you found the princess yet?! Is she safe?";
+		close;
+	case 2:
+		mes "[Unpleasent Royal Guard]";
+		mes "The monsters around the secret passage have disappeared. Now is our chance to save the princess!";
+		erasequest 1261;// Cursed Swordman
+		close;
+	}
+	mes "[Unpleasent Royal Guard]";
+	mes "I need some help; even from the likes of you.";
+	next;
+	mes "[Unpleasent Royal Guard]";
+	mes "In the middle of battle the princess disappeared while I was guarding the palace.";
+	next;
+	mes "[Unpleasent Royal Guard]";
+	mes "You run ahead and I will catch up. We must save her.";
+	next;
+	if (select( "I will help.", "This is not a good time." ) == 2) {
+		mes "[Unpleasent Royal Guard]";
+		mes "Even now all the royal guards are fighting and dying to the monsters.";
+		next;
+		mes "[Unpleasent Royal Guard]";
+		mes "We must, no matter how much fear grips our hearts; Save the Princess!";
+		close;
+	}
+	instance_create("Ghost Palace");
+	mes "[Unpleasent Royal Guard]";
+	mes "Thank you..";
+	mes "I will open the the secret passage to ^0000ffGhost Palace^000000";
+	mes "Please be ready!";
+	close;
+}
+
+dali02,40,134,5	script	Interdimensional Device#dk	PORTAL,{
+	if (BaseLevel < 120) {
+		mes "[Unpleasent Royal Guard]";
+		mes "Althought I need help with a task, you are not strong enough to assist me, noob~!";
+		next;
+		mes "[Unpleasent Royal Guard]";
+		mes "Come back after when you have done some training and are at least level 120!";
+		close;
+	}
+	mes "[Unpleasent Royal Guard]";
+	mes "I can see the secret passage into the Palace... Are you ready to save the princess?!";
```

Comment:
`I can see the secret passage into the palace. Are you ready to save the princess?!`